addons/sale_project/static/src/components/project_right_side_panel/project_right_side_panel.xml

```diff
@@ -10,34 +10,32 @@
                     <thead>
                         <tr>
-                            <th style="width: 40%">Sales Order Items</th>
-                            <th style="width: 20%; text-align: right;">Sold</th>
-                            <th style="width: 20%; text-align: right;">Delivered</th>
-                            <th style="width: 20%; text-align: right;">Invoiced</th>
+                            <th>Sales Order Items</th>
+                            <th class="text-end">Sold</th>
+                            <th class="text-end">Delivered</th>
+                            <th class="text-end">Invoiced</th>
                         </tr>
                     </thead>
                     <tbody>
                         <tr t-foreach="state.data.sale_items.data" t-as="sale_item" t-key="sale_item.id">
                             <t t-set="uom_name" t-value="sale_item.product_uom and sale_item.product_uom[1]"/>
                             <td>
                                 <t t-set="sol_name" t-value="sale_item.display_name"/>
-                                <a t-if="sale_item.action" class="o_rightpanel_button"
-                                    t-on-click="() => this.onProjectActionClick(sale_item.action)"
-                                >
+                                <a t-if="sale_item.action" class="o_rightpanel_button" href="#" t-on-click="() => this.onProjectActionClick(sale_item.action)">
                                     <t t-esc="sol_name"/>
                                 </a>
                                 <t t-else="" t-esc="sol_name"/>
                             </td>
-                            <td align="right"><t t-esc="formatValue(sale_item.product_uom_qty, uom_name)"/> <t t-esc="uom_name"/></td>
-                            <td align="right"><t t-esc="formatValue(sale_item.qty_delivered, uom_name)"/> <t t-esc="uom_name"/></td>
-                            <td align="right"><t t-esc="formatValue(sale_item.qty_invoiced, uom_name)"/> <t t-esc="uom_name"/></td>
+                            <td class="text-end"><t t-esc="formatValue(sale_item.product_uom_qty, uom_name)"/> <t t-esc="uom_name"/></td>
+                            <td class="text-end"><t t-esc="formatValue(sale_item.qty_delivered, uom_name)"/> <t t-esc="uom_name"/></td>
+                            <td class="text-end"><t t-esc="formatValue(sale_item.qty_invoiced, uom_name)"/> <t t-esc="uom_name"/></td>
                         </tr>
-                        <tr class="o_rightpanel_nohover" t-if="state.data.sale_items.total &gt; state.data.sale_items.data.length">
-                            <td>
-                                <a class="o_rightpanel_button text-bold" t-on-click="onLoadSalesOrderLinesClick">
+                        <tr class="o_rightpanel_nohover border-0" t-if="state.data.sale_items.total &gt; state.data.sale_items.data.length">
+                            <td class="border-0 text-center" colspan="100%">
```

Comment:
`100%` is not a valid  `colspan` value, this attribute takes integer numbers only.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td?retiredLocale=it#attr-colspan

Sometimes `100%` works 'cause the browser interprets it as `colspan="100"`, but setting a colpsan hither than the actual number of columns will break tables with `table-layout: fixed` ;)